dotnet-interactive.IntegrationTests/HttpClientTestExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.DotNet.Interactive.App.IntegrationTests
+{
+    internal static class HttpClientTestExtensions
+    {
+        public static async Task<HttpResponseMessage> PostJsonAsync(this HttpClient client, string requestUri,
+            JObject requestBody)
+        {
+            var content = new StringContent(requestBody.ToString(), Encoding.UTF8, "application/json");
+            var response = await client.PostAsync(requestUri, content);
+            return response;
+        }
+
+        public static async Task ShouldSucceed(this HttpResponseMessage response)
+        {
```

Comment:
The approach we took to this in dotnet/try was more specific, I believe. We should assert against the actual status code. That can be an optional parameter that defaults to 200, but a change in that status code to a different success code should still fail the test.